### PR TITLE
Adding in example project that depends on the builder

### PR DESCRIPTION
**Pull request by karthikvijayan**

### CODE REVIEW

File: examples/WORKSPACE [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	],
+	repositories = [
+         "https://maven.google.com",
+         "https://repo1.maven.org/maven2",
+     	],
+)
+
+local_repository(
+    name = "builder",
+    path = "/Users/kvijayan/git/bazel-java-builder-template",
```
Comment (guw) [inline examples/WORKSPACE]: Can you try a relative path, i.e ".."?

File: examples/BUILD [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
+load(":tree.bzl", "tree")
```
Comment (guw) [inline examples/BUILD]: This should load `mybuilder.bzl`

File: examples/tree.bzl [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Constants used
+
+INPUT_FILE = "/tmp/dependency.txt"
+OUTPUT_FILE = "/tmp/copy.txt"
+
+def _impl(ctx):
```
Comment (guw) [inline examples/tree.bzl]: This needs to move into `mybuilder.bzl`. We also need better names. `mybuilder_generator` would be better than `tree`.

**Review by guw — CHANGES_REQUESTED** [review]

File: examples/WORKSPACE [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+	"info.picocli:picocli:3.9.5",
+	],
+	repositories = [
+         "https://maven.google.com",
+         "https://repo1.maven.org/maven2",
+     	],
+)
+
+local_repository(
+    name = "builder",
```
Comment (guw) [inline examples/WORKSPACE]: For consistency this should be `mybuilder`

File: examples/WORKSPACE (hunk 1 above)
Comment (karthikvijayan) [reply examples/WORKSPACE]: We can try this tomorrow after getting it to run, just leaving it as is before execution.

File: examples/BUILD [hunk 5]
```diff
@@ -0,0 +1,12 @@
+package(default_visibility = ["//visibility:public"])
+load("//bazel-java-builder-template/mybuilder:mybuilder.bzl", "mybuilder")
```
Comment (karthikvijayan) [inline examples/BUILD]: Just doubtful if this import is in correct format?

**Pull request closed without merging**